### DISCUSSION TITLE
Is it possible to chain commands with pipe operator?

### DISCUSSION THREAD

**Discussion by havebeenfitz** [kgit2/kommand#15] [Q&A] [answered]
Hi! Basically the title is the question) As I understand, current implementation is not capable of doing something like `pwd | ls | cd` or similar. But maybe I'm missing something? 

**Some context:**

We would like to be able to trigger any command from the outside, but even if I'll parse the command string, the input of the second command (`ls` in the example above) would not be aware of the output of the first command (pwd). 

Is there a way to work around that in the current implementation?

**Comment by BppleMan** [comment, marked as answer]
You might try bash -c, using bash as the main command of Command, -c is the first parameter, followed by your external command as the second parameter

**Reply by havebeenfitz** [reply]
Thank you, that worked!
